pgmq-extension/sql/pgmq--1.5.1--1.5.2.sql

```diff
@@ -435,3 +435,117 @@ BEGIN
 
 END;
 $$ LANGUAGE plpgsql;
+
+-- detach all pgmq objects
+-- note--this will prevent `DROP EXTENSION pgmq CASCADE` from being able to drop the extension objects
+CREATE OR REPLACE FUNCTION pgmq."detach_all"()
+RETURNS TABLE(detached_object TEXT)
+LANGUAGE plpgsql
+AS $$
+DECLARE
+    queue_record RECORD;
+    detach_cmd TEXT;
+BEGIN
+    -- Only proceed if pgmq extension exists (not applicable when SQL-only)
+    IF NOT pgmq._extension_exists('pgmq') THEN
+        RAISE NOTICE 'pgmq extension does not exist, nothing to detach';
+        RETURN;
+    END IF;
+
+    -- Iterate through all queues and detach their objects (table, archive table, sequence)
+    FOR queue_record IN SELECT queue_name FROM pgmq.meta LOOP
+        BEGIN
+            detach_cmd := format('ALTER EXTENSION pgmq DROP TABLE pgmq.q_%I', queue_record.queue_name);
```

Comment:
```suggestion
            detach_cmd := format('ALTER EXTENSION pgmq DROP TABLE pgmq.%I', 'q_'  || queue_record.queue_name);
```

Reply:
This change needs to be made throughout, or else table names will render as `q_"Foo"` instead of the expected `"q_Foo"`.